### ISSUE TITLE
Fix formatting in documentation

### ISSUE THREAD

**Issue by csoneson** [iSEE/iSEE#662]
```
* checking Rd files ... NOTE
checkRd: (-1) registerAppOptions.Rd:46-48: Lost braces in \itemize; meant \describe ?
checkRd: (-1) registerAppOptions.Rd:49-51: Lost braces in \itemize; meant \describe ?
checkRd: (-1) registerAppOptions.Rd:52-54: Lost braces in \itemize; meant \describe ?
checkRd: (-1) registerAppOptions.Rd:55: Lost braces in \itemize; meant \describe ?
checkRd: (-1) registerAppOptions.Rd:56: Lost braces in \itemize; meant \describe ?
checkRd: (-1) registerAppOptions.Rd:57: Lost braces in \itemize; meant \describe ?
```

**Comment by federicomarini** [comment]
currently being addressed as a byproduct of https://github.com/iSEE/iSEE/tree/showstopper

**Comment by federicomarini** [comment]
See PR in #671

**Comment by csoneson** [comment]
Closed by #671
